docs/Tutorial_Top_Ten_3.ipynb

```diff
@@ -0,0 +1,251 @@
+{
```

Comment:
<p>I didn't know about <code>Semordnilaps</code> . So, I had to google it, and then I realized it is defined as:<code>A reverse spelling of&nbsp;palindromes</code>.</p><p> </p><p>So, I think this jargon may not be suitable for the tutorial. If there is no other way to explain this, then you may want to briefly describe <code>Semordnilaps.</code></p><p> </p><p>Also, can you provide a very simple example for TSRC. For instance, I think the array <code>np.array([1, 2, 3, 1000, 2000, -1, -2, -3])</code> contains TSRC. Correct? If yes, then this can be one simple example to show the concept. If you have a better example, please feel free to use that.</p><p> </p><p>Then, you can say something like this:</p><pre class="ql-syntax" spellcheck="false">Here, we can see that the pattern [-1, -2, -3] is the flipped, upsidedown version of pattern [1, 2, 3]. These two patterns are TSRCs.
</pre><p> </p><p>Please note that this is just an example. You may find a better example and come up with a better sentence :)</p><p> </p><p> </p><blockquote>ere is approximately 2 min from a dataset from a hip-worn accelerometer of quotidian activity from USC-HAD Database (Zhang and Sawchuk 2012)[2].</blockquote><p>I think the sampling rate is 100 per second. Right? (because, later, you use <code>m=2000</code> to find motif of length 20 seconds)</p><p> </p><p>So, you may want to mention the sampling rate here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/676/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>